test_builder/test_builder

```diff
@@ -61,6 +61,17 @@ Configuration syntax (used for configuration_prefix and configurationN values):
  'protocol_driver=bar' can be abbreviated 'bar'
  protocol_driver defaults to grpc if it is not specified.
 
+ An special parameter is extra_test_contents, which will be copied verbatim
```

Comment:
A special parameter is extra_test_contents, its value...

Reply:
done